app/src/main/kotlin/com/hippo/ehviewer/ui/reader/NavigationOverlay.kt

```diff
@@ -0,0 +1,84 @@
+package com.hippo.ehviewer.ui.reader
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.core.tween
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import eu.kanade.tachiyomi.data.preference.PreferenceValues
+import eu.kanade.tachiyomi.ui.reader.viewer.ViewerNavigation
+import eu.kanade.tachiyomi.ui.reader.viewer.navigation.DisabledNavigation
+
+@Composable
+fun NavigationOverlay(
+    navigation: ViewerNavigation,
+    invertMode: PreferenceValues.TappingInvertMode,
+    modifier: Modifier = Modifier,
+) {
+    var firstLaunch by remember { mutableStateOf(true) }
```

Comment:
Should be `rememberMutableStateInDataStore`

Reply:
Maybe we just introduce `latestGuidedNavigationOverlayType` remembered in datastore?

Reply:
> Should be `rememberMutableStateInDataStore`

This is meant to be first launch of the reader not the app, we don't want to persist it.

Reply:
But we do should show guide at first launch of app.

Reply:
> But we do should show guide at first launch of app.

And the same behaviour, guide should not disappear if user doesn't dismiss the guide but relaunch the reader

Reply:
> > But we do should show guide at first launch of app.
> 
> And the same behaviour, guide should not disappear if user doesn't dismiss the guide but relaunch the reader

https://github.com/FooIbar/EhViewer/blob/c1c6f257639fdd6313f955e1d7fac654b1b02ea3/app/src/main/kotlin/eu/kanade/tachiyomi/ui/reader/viewer/ViewerConfig.kt#L71-L74
Isn't it immediately set to false?

Reply:
Yeah, that's the origin behaviour, but we can do better